fs/config/configflags/configflags.go

```diff
@@ -79,8 +82,8 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.BoolVarP(flagSet, &fs.Config.NoCheckDest, "no-check-dest", "", fs.Config.NoCheckDest, "Don't check the destination, copy regardless.")
 	flags.BoolVarP(flagSet, &fs.Config.NoUnicodeNormalization, "no-unicode-normalization", "", fs.Config.NoUnicodeNormalization, "Don't normalize unicode characters in filenames.")
 	flags.BoolVarP(flagSet, &fs.Config.NoUpdateModTime, "no-update-modtime", "", fs.Config.NoUpdateModTime, "Don't update destination mod-time if files identical.")
-	flags.StringVarP(flagSet, &fs.Config.CompareDest, "compare-dest", "", fs.Config.CompareDest, "Include additional server-side path during comparison.")
-	flags.StringVarP(flagSet, &fs.Config.CopyDest, "copy-dest", "", fs.Config.CopyDest, "Implies --compare-dest but also copies files from path into destination.")
+	flags.StringVarP(flagSet, &compareDest, "compare-dest", "", "", "Include additional comma separated server-side paths during comparison.")
```

Comment:
Instead of parsing by hand the `:` or `;` separated string, use `flags.StringArrayVarP` which will allow the `--compare-dest` flag to be repeated. This is how the other flags work in rclone.

Reply:
> Instead of parsing by hand the `:` or `;` separated string, use `flags.StringArrayVarP` which will allow the `--compare-dest` flag to be repeated. This is how the other flags work in rclone.

@ncw 
Awesome! Thank you.